app/src/test/java/org/simple/clinic/newentry/PatientEntryInitTest.kt

```diff
@@ -0,0 +1,36 @@
+package org.simple.clinic.newentry
+
+import com.spotify.mobius.test.FirstMatchers.hasEffects
+import com.spotify.mobius.test.FirstMatchers.hasModel
+import com.spotify.mobius.test.InitSpec
+import com.spotify.mobius.test.InitSpec.assertThatFirst
+import org.junit.Test
+
+class PatientEntryInitTest {
+  private val initSpec = InitSpec(PatientEntryInit())
+  private val defaultModel = PatientEntryModel.DEFAULT
+
+  @Test
+  fun `when patient entry screen is created and colony or villages list is empty, then fetch colony or village list`() {
```

Comment:
We should not be including the details like if the color on the village list empty or not empty in the test. Instead, can we rename it to something like this?

`when screen is created, then load initial data`